salt/_states/better_kubernetes.py

```diff
@@ -0,0 +1,87 @@
+"""Management of Kubernetes objects as Salt states.
+
+This module defines two state functions: `object_present` and `object_absent`.
+Those will then simply delegate all the logic to the `metalk8s_kubernetes`
+execution module, only managing simple dicts in this state module.
+"""
+from salt.exceptions import CommandExecutionError
+
+__virtualname__ = 'metalk8s_kubernetes'
+
+
+def __virtual__():
+    if 'metalk8s_kubernetes.ping' not in __salt__:
```

Comment:
I'd expect this module to (also?) require the `better_kubernetes` module, though we'll want to rename those ;-)

Reply:
Indeed it should be `better_kubernetes` until we rename them :)